frontend/src/components/explorer/gemBrowser/Compartment.vue

```diff
@@ -114,6 +114,19 @@ export default {
       return l.join('');
     },
   },
+  metaInfo() {
+    if (!this.model || !this.compartment.name) {
+      return {};
+    }
+
+    const title = `${this.compartment.name}, Compartment in ${this.model.short_name}`;
+    const description = `The compartment ${this.compartment.name} in ${this.model.short_name} (version ${this.model.version}) consists of ${this.compartment.subsystemCount} subsystems, ${this.compartment.reactionsCount} reactions, ${this.compartment.metabolitesCount} metabolites, and ${this.compartment.genesCount} genes.`;
```

Comment:
I could not find something like `<meta name='description'` in the `<head>` section

Reply:
I could see it from the link you shared.

<img width="1201" alt="image" src="https://user-images.githubusercontent.com/423498/122209224-984be080-cea4-11eb-9409-80082993405f.png">